pkgs/development/python-modules/tiktoken/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, rustPlatform
+, setuptools-rust
+, libiconv
+, requests
+, regex
+, blobfile
+}:
+let
+  pname = "tiktoken";
+  version = "0.2.0";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-30Gj1HhJl1e1sy6uXpdlfPFZ2NnmdkBJ3Xw6u0nhtA8=";
+  };
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "openai";
    repo = pname;
    rev = "refs/tags/${version}";
    hash = "sha256-5YucJjoYAUTRy7oJ3yb2uE1UZDLIPXnV0bJIDgwGOeA=";
  };
```

We could (!) also use fetchFromGitHub - but that would be more of a style choice from me.

Reply:
I'll pass on that style change. Feel free to change it later if you feel strongly about it.